java/test/src/main/java/io/ray/test/ObjectRefTransferTest.java

```diff
@@ -0,0 +1,59 @@
+package io.ray.test;
+
+import io.ray.api.ActorHandle;
+import io.ray.api.ObjectRef;
+import io.ray.api.Ray;
+import java.util.ArrayList;
+import java.util.List;
+import org.testng.Assert;
+import org.testng.annotations.AfterClass;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.Test;
+
+public class ObjectRefTransferTest extends BaseTest {
+
+  @BeforeClass
+  public void setUpClass() {
+    System.setProperty("ray.raylet.config.num_workers_per_process_java", "1");
+  }
+
+  @AfterClass
+  public void tearDownClass() {
+    System.clearProperty("ray.raylet.config.num_workers_per_process_java");
+  }
+
+  @Test
+  public void testObjectTransfer() {
+    ObjectRef<String> objectRef = Ray.put("test");
+    List<ObjectRef<String>> data = new ArrayList<>();
+    data.add(objectRef);
+
+    ActorHandle<RemoteActor> handle = Ray.actor(RemoteActor::new).remote();
+    String result = handle.task(RemoteActor::get, data).remote().get();
+    Assert.assertEquals(result, "test");
+    handle.kill();
```

Comment:
You don't need to kill the actor manually. `Ray.shutdown()` is called in `BaseTest`.

Reply:
BTW, we also run tests under single-process mode. In this mode, the `kill` API is not available.

Reply:
updated